itomate/itomate.py

```diff
@@ -9,7 +9,7 @@
 import yaml
 
 default_config = 'itomate.yml'
-version = '0.3.6'
```

Comment:
@kamranahmedse Not sure how you have been doing versioning, this PR is ready to go though if updating the version this way is fine

Reply:
@kamranahmedse Bumping this